tools/cardano-sl-tools.cabal

```diff
@@ -140,13 +128,7 @@ executable cardano-wallet-hs2purs
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
 
-  if flag(with-wallet) && flag(with-web)
-    buildable:         True
-  else
-    buildable:         False
-
-  if flag(with-web)
-    cpp-options: -DWITH_WEB
+  buildable:         False
```

Comment:
This is wrong, we need to build `cardano-wallet-hs2purs`. I propose to move this executable to `cardano-sl-wallet`.